pkg/cmd/registry/create/create.go

```diff
@@ -156,7 +157,16 @@ func runCreate(opts *options) error {
 
 	opts.Logger.Info(icon.SuccessPrefix(), opts.localizer.MustLocalize("registry.cmd.create.info.successMessage"))
 
-	compatibleEndpoints := registrycmdutil.GetCompatibilityEndpoints(response.GetRegistryUrl())
+	registry, _, err := serviceregistryutil.GetServiceRegistryByID(opts.Context, conn.API().ServiceRegistryMgmt(), response.GetId())
+	if err != nil {
+		return err
+	}
+
+	if err = dump.Formatted(opts.IO.Out, opts.outputFormat, registry); err != nil {
+		return err
+	}
+
+	compatibleEndpoints := registrycmdutil.GetCompatibilityEndpoints(registry.GetRegistryUrl())
```

Comment:
Trick is that this is async call so when I run this pointed to local it can still return nil 

```suggestion
      if registry.GetRegistryUrl() != nil {
	compatibleEndpoints := registrycmdutil.GetCompatibilityEndpoints(registry.GetRegistryUrl())
      } else {
          // To get list of compatible endpoints please run `describe` command
      }
```